pkg/api/openapi/minder/v1/minder.swagger.json

```diff
@@ -3902,6 +4014,10 @@
           "type": "string",
           "description": "subject is the subject to which the role is assigned."
         },
+        "displayName": {
```

Comment:
I don't know if we are in the position to release a breaking change right now, but I think it would make more sense to nest displayName under subject, somehow like this:

```js
{
  "subject": {
    "id": // what is now called subject
    "displayName": // etc
  }
}
```

also I think "user" could be a simpler naming, at least as long as users are the only possible subjects for role assignment (maybe groups or projects will be also possible in the future?)

Reply:
I'm not looking to make breaking API changes.  Sorry!  (Yeah, some of the API naming is very technical rather than friendly.  /shrug)